homeassistant/components/nam/const.py

```diff
@@ -122,13 +125,15 @@
         key=ATTR_SDS011_P1,
         name=f"{DEFAULT_NAME} SDS011 Particulate Matter 10",
         native_unit_of_measurement=CONCENTRATION_MICROGRAMS_PER_CUBIC_METER,
+        device_class=DEVICE_CLASS_PM10,
```

Comment:
When a device_class is defined, the icon should not be defined.

Reply:
Even if device class doesn't define an icon?

Reply:
Yes.
What device classes do not define an icon?

Reply:
The icons aren't displayed because `dev` branch doesn't use the latest frontend version yet, I missed that. So everything is fine, I remove icons.